src/main/resources/db/changelog/changesets/21052021_initial_schema.yaml

```diff
@@ -0,0 +1,109 @@
+databaseChangeLog:
+  - changeSet:
+      id: "1"
+      author: kfeldman, swessel, zarsky
+      changes:
+        - sql:
+            sql: "CREATE TYPE token_type_enum AS ENUM ('document', 'code')"
```

Comment:
I would call this `document_token` and `access_token` (or just `document` and `access` but plain `access` feels weird)